manager/controllers/app/fybrik_notebook_test.go

```diff
@@ -146,15 +146,15 @@ func TestS3Notebook(t *testing.T) {
 		return application.Status.Ready
 	}, timeout, interval).Should(Equal(true))
 
-	blueprintNamespace := utils.GetBlueprintNamespace()
-	fmt.Printf("blueprint namespace notebook test: %s\n", blueprintNamespace)
+	dataAccessModuleNamespace := utils.GetDataAccessModuleNamespace()
+	fmt.Printf("data access module namespace notebook test: %s\n", dataAccessModuleNamespace)
 
 	// Forward port of arrow flight service to local port
-	svcName := strings.Replace(application.Status.AssetStates["fybrik-notebook-sample/data-csv"].Endpoint.Hostname, "."+blueprintNamespace+".svc.cluster.local", "", 1)
+	svcName := strings.Replace(application.Status.AssetStates["fybrik-notebook-sample/data-csv"].Endpoint.Hostname, "."+dataAccessModuleNamespace+".svc.cluster.local", "", 1)
```

Comment:
Unrelated to this change, the .svc.cluster.local should be removed. This is the cluster domain name and is configurable. So may not be svc.cluster.local. Besides it is not needed when accessing the endpoint.

Reply:
Removed.